<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Setting_Tip">
     <property name="text">
      <string>设置页面</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="TabW_Main">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NormalTab">
      <attribute name="title">
       <string>一般设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="SA_NT">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="SA_NT_WidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1166</width>
            <height>475</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_NT_TotoalStyle_Tip">
             <property name="text">
              <string>风格</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="GL_NT" rowstretch="0,0,0,0" columnstretch="1,4">
             <item row="0" column="1">
              <widget class="QFontComboBox" name="FontCB_NT"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_NT_FontSize_Tip">
               <property name="text">
                <string>字体大小</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="HL_NT_Style">
               <item>
                <widget class="QCheckBox" name="checkBox_NT_Light">
                 <property name="text">
                  <string>浅色</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_NT_Dark">
                 <property name="text">
                  <string>深色</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_NT_Style_Tip">
               <property name="text">
                <string>界面风格</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_NT_Font_Tip">
               <property name="text">
                <string>字体</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_NT_FontWeight_Tip">
               <property name="text">
                <string>字体粗细</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_NT_FontSize">
               <property name="value">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBox_NT_FontWeight">
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="VS_NT">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>374</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SpecialTab">
      <attribute name="title">
       <string>特殊设置</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HL_Control" stretch="3,1,1,1,1,1,1,1">
     <item>
      <spacer name="HS_Contrrol">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Confirm">
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Apply">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ApplyPage">
       <property name="text">
        <string>应用该页面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Reset">
       <property name="text">
        <string>全部重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ResetPage">
       <property name="text">
        <string>该页面重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CancelPage">
       <property name="text">
        <string>取消修改该页面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
